source/features/default-to-rich-diff.tsx

```diff
@@ -0,0 +1,35 @@
+import select from 'select-dom';
+import features from '../libs/features';
+
+const observer = new MutationObserver(([{addedNodes}]) => {
+	for (const node of addedNodes) {
+		if (node instanceof Element) {
+			if (node.matches('.js-diff-progressive-container')) {
```

Comment:
@bfred-it I wasn't able to use both conditions in the same if because we would get errors when using a non-Element node as BaseElement of `setRichDiff`.